addons/popochiu/engine/objects/character/popochiu_character.gd

```diff
@@ -594,6 +594,9 @@ func play_animation(animation_label: String, animation_fallback := 'idle'):
 			[script_name]
 		)
 		return
+	
+	if $AnimationPlayer.get_animation_list().is_empty():
```

Comment:
This smells like a bugfix sneaked into this PR, right?

Reply:
More than a bugfix, I see it as a small UX improvement. Many messages like this appear in the Output:

![image](https://github.com/carenalgas/popochiu/assets/4536477/7bb340e1-644f-44f6-b75e-ca6cc9b8e5f2)

They can be annoying during the early stages of development if animations are not yet ready, or if one has characters without animations.

This makes sure the logic about triggering the right animation is not done if there are no animations in the character.